src/main/java/org/checkerframework/specimin/TargetMethodFinderVisitor.java

```diff
@@ -478,6 +487,7 @@ public void updateUsedElementWithPotentialFieldNameExpr(NameExpr expr) {
       String classFullName = exprDecl.asField().declaringType().getQualifiedName();
       updateUsedClassWithQualifiedClassName(classFullName);
       usedMembers.add(classFullName + "#" + expr.getNameAsString());
+      updateUsedClassWithPossibleTypeArgs(exprDecl.getType());
```

Comment:
I notice that every place you've added a call to this method, there is a call to `updateUsedClassWithQualifiedClassName` immediately beforehand. Is it possible to combine the two methods? As it is, I think it would be easy for someone to call one without calling the other (and it's difficult for me to verify in a code review that you haven't missed somewhere that `updateUsedClassWithQualfiedClassName` is called, since I'm only looking at the diff)

Reply:
It's not always the case that these two methods appear together. These two methods only appear together when we are updating the list of used classes based on the types of some used members.

When updating the list of used classes based on the used members themselves, we only call `updateUsedClassWithQualifiedClassName`.

For example, given a method declaration, we will add the class containing that method to the list of used classes. In that case, only `updateUsedClassWithQualifiedClassName` is called. When we update the list of used classes based on the return type of that method declaration, then we will call `updateUsedClassWithQualifiedClassName` and `updateUsedClassWithPossibleTypeArgs` together.

Reply:
Please update the documentation of both methods to explain this.